lighty-modules/lighty-gnmi/lighty-gnmi-sb/README.md

```diff
@@ -1,6 +1,15 @@
 # gNMI south-bound module
-is south-bound lighty module which manages connection with gNMI target.
-This module implements functionality to make CRUD operations on gNMI target.
+is south-bound lighty module which manages connection with gNMI targets. This module implements functionality to make
+CRUD operations on multiple gNMI targets. Which make it easy to read and manipulate with data in gNMI devices.
+
+Lighty gNMI augment ODL network-topology model with [gnmi-topology](../../../lighty-models/lighty-gnmi-models/lighty-gnmi-topology-model/src/main/yang/gnmi-topology.yang)
+model. This allows to configure and register a specific gNMI device as a [NETCONF node](https://docs.opendaylight.org/projects/netconf/en/latest/user-guide.html#spawning-new-netconf-connectors).
```

Comment:
I don't think it is a good idea to explain it referencing NETCONF .. The link contains RESTCONF commands, which are not relevant in this README since it does not use RESTCONF at all. I would delete this or rewrite it accordingly.

Reply:
done